src/main/java/net/openhft/chronicle/threads/LongPauser.java

```diff
@@ -132,9 +154,8 @@ public void pause(long timeout, @NotNull TimeUnit timeUnit) throws TimeoutExcept
         }
 
         // If a finite timeout is given, check if it's exceeded and throw a TimeoutException if it is
-        if (timeout < Long.MAX_VALUE) {
-            if (firstPauseNS + timeUnit.toNanos(timeout) - now < 0)
-                throw new TimeoutException();
+        if (timeout < Long.MAX_VALUE && (firstPauseNS + timeUnit.toNanos(timeout) - now < 0)) {
```

Comment:
I think we should mention this in the issue so it appears in release notes

Reply:
This isn't a functional change, it's removing an unneeded nested condition. It could be reverted if this isn't clearer.